debian/xenial/kubelet/debian/control

```diff
@@ -1,16 +1,16 @@
 Source: kubelet
 Section: misc
 Priority: optional
-Maintainer: Mike Danese <mikedanese@google.com>
+Maintainer: Kubernetes Authors <kubernetes-dev@google.com>
 Build-Depends: debhelper (>= 8.0.0)
 Standards-Version: 3.9.4
 Homepage: https://kubernetes.io
 Vcs-Git: https://github.com/kubernetes/kubernetes.git
 Vcs-Browser: https://github.com/kubernetes/kubernetes
 
 Package: kubelet
-Architecture: amd64
-Depends: docker-engine (= 1.11.2-0~xenial), iptables (>= 1.4.21), k8s-cni (= 0.0.1-0),
- ${shlibs:Depends}, ${misc:Depends}
+Architecture: {{ .Arch }}
+Depends: docker-engine (>= 1.11.2-0~xenial), iptables (>= 1.4.21), kubernetes-cni (>= 0.0.1-0),
```

Comment:
Can you wrap this in a `{{ default "docker-engine (>= 1.11.2-0~xenial), iptables (>= 1.4.21), kubernetes-cni (>= 0.0.1-0), ${misc:Depends}" .DependsOverride }}` template field? Then it's easy to override the deps for a given package and arch in a generic manner...